src/fairchem/core/common/relaxation/ase_utils.py

```diff
@@ -80,6 +80,7 @@ def __init__(
         max_neighbors: int = 50,
         cpu: bool = True,
         seed: int | None = None,
+        only_output: list[str] | None = None,
```

Comment:
nit, better to have `only_output` as a tuple, since its not meant to be changed at all in the code